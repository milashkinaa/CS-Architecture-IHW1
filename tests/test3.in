7
-11
-9
-8
-5
0
1
4
