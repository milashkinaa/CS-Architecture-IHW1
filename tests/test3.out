Введите размер массива=a[0] = a[1] = a[2] = a[3] = a[4] = a[5] = a[6] = Получившаяся последовательность: -11 -9 -8 -5 0 1 4 
